src/scrot.c

```diff
@@ -726,9 +725,8 @@ static char *scrotGetWindowName(Window window)
             &clsHint);
 
     if (status != 0) {
-        windowName = estrdup(clsHint.res_class);
+        windowName = clsHint.res_class;
```

Comment:
NOTE: A bit weird that the function name is `GetWindowName` but it's returning the *class* instead.